packages/airnode-node/src/workers/local-gateways/validation.ts

```diff
@@ -102,6 +119,164 @@ export function verifyHttpSignedDataRequest(
   return { success: true, encodedParameters, endpointId: validEndpointId };
 }
 
+const beaconSchema = z.object({
+  airnodeAddress: z.string(),
+  endpointId: z.string(),
+  encodedParameters: z.string(),
+  // Fields encodedValue, timestamp and signature are optional as they might be missing if we have no data for some of the beacons.
+  // This is fine as we would ignore those but we still need to know how many beacons are in the beacon set and Airnode
+  // addresses to derive the data feed ID.
+  encodedValue: z.string().optional(),
+  timestamp: z.string().optional(),
+  signature: z.string().optional(),
+});
+
+type Beacon = z.infer<typeof beaconSchema>;
+interface BeaconDecoded extends Required<Beacon> {
+  decodedValue: ethers.BigNumber;
+}
+
+export const signOevDataBodySchema = z.object({
+  chainId: z.number().positive(),
```

Comment:
(nit) You also specify .min(1) in the OpenAPI spec.

Reply:
Yes, because in the OpenAPI spec I can't say `positive`.